composer.json

```
@@ -6,27 +6,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
+        "drupal/core": "^8.9 || ^9.1",
         "drupal/sparql_entity_storage": "^1.0.0-alpha8",
         "easyrdf/easyrdf": "1.0.0 as 0.9.1",
-        "php": ">=7.2"
+        "php": ">=7.3"
     },
     "require-dev": {
         "composer/installers": "^1.5",
         "consolidation/robo" : "~1.4",
         "cweagans/composer-patches": "~1.0",
-        "drupal-composer/drupal-scaffold": "^2.5.2",
+        "drupal/core-composer-scaffold": "^8.9 || 9",
         "drupal/config_devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0",
-        "guzzlehttp/guzzle": "~6.3",
+        "drush/drush": "~10.0",
         "nikic/php-parser": "^3.1.5",
         "openeuropa/behat-transformation-context" : "~0.1",
-        "openeuropa/code-review": "~1.5",
-        "openeuropa/drupal-core-require-dev": "^8.9.13",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0",
-        "symfony/dom-crawler": "~3.4"
+        "openeuropa/code-review": "dev-EWPP-372",
+        "openeuropa/drupal-core-require-dev": "^8.9.13 || ^9.1.6",
```

This needs to be the same as what's in `"require"`, we let composer resolve versions.
```suggestion
        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
```